extensions/azurePublishNew/src/components/AzureProvisionWizard.tsx

```diff
@@ -43,7 +45,14 @@ export const AzureProvisionWizard = () => {
   }, [provisionAction]);
 
   return (
-    <RecoilRoot>
+    <RecoilRoot
```

Comment:
The current extension has initialization of merging incoming state with defaults.  We should replicate that logic here when initializing the recoil state.  It might be good to factor it into a hook and then use that hook to get the initial state for the recoil root.